src/index.js

```diff
@@ -1,5 +1,8 @@
 import React from "react";
 import ReactDom from "react-dom";
+// Necessary import to support async/await
```

Comment:
Is this only for compatibility?

Reply:
Yeah sort of. It imports a polyfill which Babel uses to allow async await on older browsers IIRC